### PR TITLE
Hanna edits food

### PR DESCRIPTION
**Pull request by hlandrus**

### CODE REVIEW

**Comment by hlandrus** [comment]
Added in more links. More to come!

File: dining_guide.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Seattle Dining Guide
 .. contents::
    :local:
 
-U-District (Conference Vicinity)
+U-District (Conference Vicinity Lunch)
 --------------------------------
```
Comment (Mariatta) [inline dining_guide.rst]:
```suggestion
--------------------------------------
```

Comment (Mariatta) [reply dining_guide.rst]: Just need to make the dashes same length as the previous line, so Travis will pass :)

Comment (hlandrus) [reply dining_guide.rst]: Sounds good. Thank you.

File: dining_guide.rst [hunk 2]
```diff
@@ -75,21 +95,54 @@ and other nightlife. To get to Capitol Hill via public transit you can take the
...
 (0.3 mile / 0.5 km walk from Westlake Station) or in Belltown on 1st and 2nd Ave 
 (0.5 miles / 0.8 km). 
 
+Short list:
+
+- `Yard House, americans + beer <https://www.yardhouse.com/home>`_ all ages lots of space, 20 minute public transit
+
+- `Suite 410, fancy cocktails <http://suite410bar.com/>`_ smaller space 20 minute public transit
+
+- `Shorty's, arcade bar with hotdogs http://www.shortydog.com/>`_ 25 minute public transit
```
Comment (Mariatta) [inline dining_guide.rst]:
```suggestion
- `Shorty's, arcade bar with hotdogs <http://www.shortydog.com/>`_ 25 minute public transit
```

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**